agency/src/services/schema/schema.class.ts

```diff
@@ -8,20 +8,21 @@ import {
   AriesCredDefServiceRequest,
   CredDefServiceResponse,
 } from '../../models/credential-definition';
-import { SchemaServiceAction, ServiceType } from '../../models/enums';
+import { CredDefServiceAction, EndorserServiceAction, SchemaServiceAction, ServiceType } from '../../models/enums';
 import {
   AriesSchema,
   AriesSchemaServiceRequest,
   SchemaServiceModel,
 } from '../../models/schema';
 import { IssuerServiceParams } from '../../models/service-params';
+import { deferServiceOnce } from '../../utils/sleep';
 import { AriesAgentData } from '../aries-agent/aries-agent.class';
 
 interface ServiceOptions { }
 
 export class Schema implements ServiceSwaggerAddon {
   app: Application;
-  options: ServiceSwaggerOptions;
+  options: ServiceOptions;
```

Comment:
I believe this should be reverted to ensure the Swagger UI is generated correctly (it is the interface that exposes the `model` properties used to define the schema).